README.md

```diff
@@ -30,15 +30,16 @@ cd ./standards-registry/ui
 Create a `.env` file containing the following content:
 
 ```
-CKAN_URL=https://manage.test.nhs.marvell-consulting.com/api/3/action
+touch .env.local
+echo "CKAN_URL=https://manage.test.nhs.marvell-consulting.com/api/action" >> .env.local
 ```
 
 ### Running the server
 
 Then run the following commands in the `./ui` directory:
 
 ```
-npm ci
+npm install
```

Comment:
What's the reasoning behind this change?

`npm ci` is objectively preferable to `npm install` in this context.

Reply:
I think Karwai had a different version of npm installed and this threw as a result. `npm i` is a bit more forgiving, but, yes actually `npm ci` is preferable

Reply:
If Karwai is running an npm version that doesn't have `ci` she must be on some **OLD** node version, so I would recommend she updates.